.github/workflows/ubuntu_22.yml

```diff
@@ -375,6 +375,16 @@ jobs:
       runner: 'ubuntu-22.04-16-cores'
       model_scope: 'nightly_scope2'
 
+  JAX_Models_Tests_Precommit:
+    name: JAX/Flax Models tests
+    if: fromJSON(needs.smart_ci.outputs.affected_components).JAX_FE.test
+    needs: [ Docker, Build, Smart_CI, Openvino_tokenizers ]
+    uses: ./.github/workflows/job_jax_models_tests.yml
+    with:
+      runner: 'aks-linux-8-cores-16gb'
+      model_scope: 'precommit'
+      container: '{"image": "${{ fromJSON(needs.docker.outputs.images).ov_build.ubuntu_22_04_x64 }}", "volumes": ["/mount:/mount"]}'
```

Comment:
I suppose it should be test image?

Reply:
propose to do this in separate PR by GHA team since it relates to all FEs

Reply:
Created ticket 152350